content/en/docs/concepts/services-networking/dns-pod-service.md

```diff
@@ -306,7 +306,7 @@ When the Pod above is created, the container `test` gets the following contents
 in its `/etc/resolv.conf` file:
 
 ```
-nameserver 1.2.3.4
+nameserver 192.0.2.1
```

Comment:
please add a comment like `# 192.0.2.1 is an example IP`.

Reply:
>       Lines that contain a semicolon (;) or hash character (#) in the
>       first column are treated as comments.

For `resolv.conf`, that comment does need to be on a line by itself.